app/src/main/java/com/example/topmovies/viewmodel/MovieViewModel.kt

```diff
@@ -0,0 +1,74 @@
+package com.example.topmovies.viewmodel
+
+import android.content.SharedPreferences
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.topmovies.model.Movie
+import com.example.topmovies.model.MovieDetails
+import com.example.topmovies.model.MovieObject
+import com.example.topmovies.repository.MovieRepository
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+val TAG = MovieViewModel::class.simpleName
+
+class MovieViewModel constructor(
+    private val repository: MovieRepository,
+    private val sharedPref: SharedPreferences
+) : ViewModel() {
+    
+    private val _movies = MutableLiveData<List<Movie>?>()
+    val movies: MutableLiveData<List<Movie>?> = _movies
```

Comment:
Why that's nullable?

Reply:
Because in _movies we post value response.body(), and method body() mark like Nullable

Reply:
I need this for running my code, but it is the wrong solution, I have just fixed it